proxy/envoy/config.go

```diff
@@ -322,6 +324,20 @@ func buildOutboundRoutes(instances []*model.ServiceInstance, services []*model.S
 	return httpConfigs, tcpConfigs
 }
 
+func buildSslContext(hostname string, context *ProxyContext) *SslContext {
+	mesh := context.MeshConfig
+	serviceAccounts := context.Discovery.GetIstioServiceAccounts(hostname)
+	if mesh.AuthMode == "istio-tls" {
+		return &SslContext {
+			CertChainFile:          mesh.AuthConfigPath + "/cert_chain",
```

Comment:
FYI the file name used in istio/auth is slightly different from here. See https://github.com/istio/auth/blob/master/controller/secret.go#L116